load.php

```diff
@@ -236,6 +236,13 @@ function perflab_can_load_module( $module ) {
 		return true;
 	}
 
+	$standalone_plugin_config = perflab_get_standalone_plugins_config();
+	foreach ( $standalone_plugin_config as $slug => $constant ) {
+		if ( $module === $slug && defined( $constant ) ) {
+			return new WP_Error( 'standalone_plugin_activated', __( 'The module cannot be managed with Performance Lab since it is already active as a standalone plugin.', 'performance-lab' ) );
+		}
+	}
```

Comment:
A `foreach` here is not good for performance and shouldn't be necessary. You only need to check for the slug.
```suggestion
	if ( isset( $standalone_plugin_config[ $module ] && defined( $standalone_plugin_config[ $module ] ) ) {
		return new WP_Error( 'standalone_plugin_activated', __( 'The module cannot be managed with Performance Lab since it is already active as a standalone plugin.', 'performance-lab' ) );
	}
```